### DISCUSSION TITLE
Can gocryptfs.conf be safely included in a backup?

### DISCUSSION THREAD

**Discussion by urathe** [rfjakob/gocryptfs#886] [Q&A] [answered]
I'm using gocryptfs in reverse mode to create an encrypted view of (local) data. This encrypted view is then backed up to an unsafe (remote) file system using rsync.

Can I safely include the file gocryptfs.conf in the backup to the remote location, or should I exclude it from the backup? I understand gocryptfs.conf contains the master key, encrypted with the password. I'm wondering whether access to this file would be helpful to an attacker trying to break the file system encryption. (Assume an adequate password of 25+ random characters from a large character set.)

I couldn't find advice on this issue in the documentation and discussions on github. Apologies if the question is a bit basic.

**Comment by rfjakob** [comment, marked as answer]
Yes, as long as the password is strong ( https://github.com/rfjakob/gocryptfs/wiki/Password-Strength ), and yours looks strong, you can safely include it in the backup.

**Reply by urathe** [reply]
Thank you very much. Great tool!
